doc/about/api/index.md

```diff
@@ -19,12 +19,13 @@ Exceptions to the previous rule:
      `detail` or if it is private to a class type, you are not allowed to depend upon it any way. More generally,
```

Comment:
Nicht sicher, mein "Sprachgefühl" sagt mir, dort muss noch ein "in" oder ähnliches rein.
```suggestion
     `detail` or if it is private to a class type, you are not allowed to depend upon it in any way. More generally,
```